qiskit_experiments/framework/base_experiment.py

```diff
@@ -479,9 +478,27 @@ def set_analysis_options(self, **fields):
         )
         self.analysis.options.update_options(**fields)
 
-    def _postprocess_transpiled_circuits(self, circuits: List[QuantumCircuit], **run_options):
-        """Additional post-processing of transpiled circuits before running on backend"""
-        pass
+    def _transpile(
+        self, circuits: List[QuantumCircuit], backend: Backend, **transpile_options
+    ) -> List[QuantumCircuit]:
+        """Transpile the experiment circuits.
+
+        This function is used to transpile the experiment circuits before sending them to
+        the backend. It can be overridden to define custom transpilation.
+        """
+        transpiled = transpile(circuits, backend, **transpile_options)
+
+        # TODO remove this deprecation after 0.3.0 release
+        if hasattr(self, "_postprocess_transpiled_circuits"):
```

Comment:
Good catch. This is for custom experiment code that has already built by the community?

Reply:
Yes.